README.rst

```diff
@@ -33,27 +32,27 @@ Documentation and a simple tutorial can be found in the docs subdirectory and on
 Installation
 ------------
 
-`pyblock` can be used simply by adding to `$PYTHONPATH`.  Alternatively, it can be
+``pyblock`` can be used simply by adding to ``$PYTHONPATH``.  Alternatively, it can be
 installed using distutils by running:
 
 ::
 
     $ pip install /path/to/pyblock
 
-where `/path/to/` is the (relative or absolute) path to the directory containing
-`pyblock`.  To install an editable version (useful for development work) do:
+where ``/path/to/`` is the (relative or absolute) path to the directory containing
+``pyblock``.  To install an editable version (useful for development work) do:
 
 ::
 
     $ pip install -e /path/to/pyblock
```

Comment:
This no longer works -- pip only supports editable installs for a setup.py-based workflow.

Reply:
Should I remove this line from the README?

Reply:
```
pip install flit
flit install --symlink
```

works fine?